calico-cloud/observability/elastic/flow/aggregation.mdx

```diff
@@ -88,6 +86,7 @@ Before [changing the default aggregation level](../../../reference/resources/fel
 
 - Although any change in aggregation level affects flow log volume, lowering the aggregation number (especially to `0` for no aggregation) will cause significant impacts to log storage. If you allow more flow logs, ensure that you provision more log storage.
 - Verify that the parameters that you want to see in your aggregation level are not already [filtered](filtering.mdx).
+- Note that currently the numeric levels must be used instead of the names in configuration.
```

Comment:
```suggestion
- Note that the numeric values must be used instead of the names in configuration.
```
They correspond with levels, but when they're entered in the config, perhaps better to call 'value'.